### ISSUE TITLE
Error: ENOENT: no such file or directory when running mirror --live

### ISSUE THREAD

**Issue by dwiel** [holepunchto/drives#8]
I was running some tests and ran into this problem:

```
z:src zdwiel$ drives mirror serial_to_pubsub/ 7bbiwns7mju7htdncgdgsomdy1toswqo5muyfjd7ut81wd9zwwfy --live
✔ Total files: 6 (+0 -0 ~0)

+ /n~ 6Bnode:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open '/Users/zdwiel/src/serial_to_pubsub/n~'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/zdwiel/src/serial_to_pubsub/n~'
}

Node.js v19.7.0
```

looks like maybe the file n~ disappeared before drives had a chance to open it? I wasn't doing anything crazy, just creating and saving files with vi and deleting files from the command line.

**Comment by LuKks** [comment]
Yes, it's a `mirror-drive` bug: https://github.com/holepunchto/mirror-drive/pull/15

**Comment by dwiel** [comment]
Cool, it looks I can work around this with some retry logic around the shell command itself, assuming the error isn't too frequent. Does that sound right? Can also wait for the fix in `mirror-drive`

**Comment by LuKks** [comment]
It's somewhat frequent depending on the editor, this happens due the temporal file that editors create and very instantly delete

I created a `drives` branch with the specific `mirror-drive` branch, try re-installing like this until fix is released:
`npm i holepunchto/drives#fix-enoent`

**Comment by dwiel** [comment]
thanks, I'll give that a shot
